src/test.rs

```diff
@@ -428,3 +428,12 @@ fn test_reentrant() {
     sender.send(null.clone()).unwrap();
     assert_eq!(null, receiver.recv().unwrap());
 }
+
+#[test]
+fn test_large_send() {
```

Comment:
We don't use `test_` prefixes in Rust...

BTW, is there some specific reason why you put this test in `src/test.rs` rather than `src/platform/test.rs`, where the existing bid data tests are?

(This would make it easier to just copy the existing `big_data()` test as `huge_data()` or so, thus addressing the problems pointed out below...)